auto3dseg/algorithm_templates/dints/scripts/train.py

```diff
@@ -116,14 +116,146 @@ def __call__(self, val_acc):
             self.counter = 0
 
 
+_libcudart = ctypes.CDLL("libcudart.so")
+# Set device limit on the current device
+# cudaLimitMaxL2FetchGranularity = 0x05
+p_value = ctypes.cast((ctypes.c_int * 1)(), ctypes.POINTER(ctypes.c_int))
+_libcudart.cudaDeviceSetLimit(ctypes.c_int(0x05), ctypes.c_int(128))
+_libcudart.cudaDeviceGetLimit(p_value, ctypes.c_int(0x05))
+assert p_value.contents.value == 128
+
+torch.backends.cudnn.benchmark = True
+torch.set_float32_matmul_precision("high")
+
+
+CONFIG = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {"monai_default": {"format": DEFAULT_FMT}},
+    "loggers": {
+        "monai.apps.auto3dseg.auto_runner": {"handlers": ["file", "console"], "level": "DEBUG", "propagate": False}
+    },
+    "filters": {"rank_filter": {"{}": "__main__.RankFilter"}},
+    "handlers": {
+        "file": {
+            "class": "logging.FileHandler",
+            "filename": "runner.log",
+            "mode": "a",  # append or overwrite
+            "level": "DEBUG",
+            "formatter": "monai_default",
+            "filters": ["rank_filter"],
+        },
+        "console": {
+            "class": "logging.StreamHandler",
+            "level": "INFO",
+            "formatter": "monai_default",
+            "filters": ["rank_filter"],
+        },
+    },
+}
+
+
+class EarlyStopping:
```

Comment:
There is another class with the same name "EarlyStopping" in line 93.

Reply:
I have removed any instances of duplication.